model/custom_file.go

```diff
@@ -2,14 +2,18 @@ package model
 
 import (
 	"fmt"
+	"strings"
+	"text/template"
 
 	"github.com/kubernetes-incubator/kube-aws/gzipcompressor"
+	"stash.hcom/run/kat/pkg/logger"
```

Comment:
Probably you mistakenly introduced an another logger that is not in the project deps?